lbry/extras/daemon/daemon.py

```diff
@@ -3123,7 +3123,10 @@ def jsonrpc_channel_list(self, account_id=None, wallet_id=None, page=None, page_
                     f"Use --allow-duplicate-name flag to override."
                 )
 
-        file_path = await self._video_file_analyzer.verify_or_repair(validate_file, optimize_file, file_path)
+        try:
+            file_path = await self._video_file_analyzer.verify_or_repair(validate_file, optimize_file, file_path)
+        except ValueError:
+            pass  # it's not a video file
```

Comment:
This isn't a very ergonomic API if it throws an exception under arguably normal circumstances (a file is not a video). Can this be handled internally in `verify_or_repair`? You can just return a  boolean from `verify_or_repair` if anything was actully done (or False if nothing was done / it was not a video, etc).

Reply:
If the mime type was determined before this point that would help a lot. As it is, the method is used both here and in the check_video script. However, the check_video script handles this exception in a much different fashion.

Reply:
mime type should be available after these lines:

https://github.com/lbryio/lbry-sdk/blob/4b87cb45eef1c5b661c96462febb34a50ddd3523/lbry/extras/daemon/daemon.py#L3127-L3128

Reply:
https://github.com/lbryio/lbry-sdk/blob/4b87cb45eef1c5b661c96462febb34a50ddd3523/lbry/schema/claim.py#L239-L245

Reply:
but... `verify_or_repair` shouldn't rely on that to the point of blowing up. I think in almost all circumstances calling `verify_or_repair` shouldn't throw an error unless `validate_file` is true and the file is not valid.

Reply:
So if I add a boolean parameter for "ignore non-video files" that would be better than the try/except?

Reply:
yes, sounds good

Reply:
Done